src/meetbot/features/command-help.ts

```diff
@@ -0,0 +1,40 @@
+import { Bot } from '..';
+import { postToChatJob } from '../pptr-helpers';
+import fs = require('fs');
+const Comments = require('parse-comments');
+const comments = new Comments();
+
+
+function commandParser(): CommandHelp[] {
+	var commandHelp: CommandHelp[] = []
+	const files = fs.readdirSync(__dirname);
+	for (const f of files) {
+		if (f.endsWith('.js') && f !== 'index.js' && f.startsWith('command-')) {
+			const ast = comments.parse(fs.readFileSync(`${__dirname}/${f}`, { encoding: 'utf8' }));
+			commandHelp.push({
+				"command": "/" + f.replace('command-', '').replace('.js', ''),
+				"description": ast[0].description,
+			})
+		}
+	}
```

Comment:
```suggestion
	const files = fs.readdirSync(__dirname);
	const commandRegExp = new RegExp('command-(.*).js');
	for (const f of files) {
		if (f !== 'index.js' && f.match(commandRegExp)) {
			const ast = comments.parse(fs.readFileSync(`${__dirname}/${f}`, { encoding: 'utf8' }));
			commandHelp.push({
				"command": "/" + f.match(commandRegExp)[1],
				"description": ast[0].description,
			})
		}
	}
```

Reply:
Not sure if using regex is cleaner, up to you :P
(haven't tested it btw)

Reply:
Oh yeah, Regex and I have a very toxic past so I often forget about it. I will try to make things work with it here.